app/[locale]/page.tsx

```diff
@@ -37,18 +37,18 @@ export default async function Home({
           {[
             {
               image:
-                'https://nitkkr.ac.in/wp-content/uploads/2023/11/IMG20220903190255-1-scaled.jpg',
+                'https://isac-nitkkr-public.s3.ap-south-1.amazonaws.com/slideshow/1.png',
               title: 'NIT KKR deemed the First Ever NIT With All Green Campus!',
               subtitle:
                 'Over 900 Acres of green foliage planted alongside the campus walls, the campus of the esteemed...',
             },
             {
               image:
-                'https://nitkkr.ac.in/wp-content/uploads/2022/01/24131961_285405678647849_426967072086000359_o.jpg',
+                'https://isac-nitkkr-public.s3.ap-south-1.amazonaws.com/slideshow/2.png',
             },
             {
               image:
-                'https://nitkkr.ac.in/wp-content/uploads/2022/12/IMG_0496-scaled.jpg',
+                'https://isac-nitkkr-public.s3.ap-south-1.amazonaws.com/slideshow/3.png',
```

Comment:
```suggestion
                'slideshow/3.png',
```
The base url is prefixed automatically

Reply:
checking

Reply:
fixed @GetPsyched